build-utils/reportdeploy.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set +x
+current_branch() {
+  echo $(git branch --contains HEAD | grep -E '^\*' | grep -Eo '\b\w.*' );
+}
+short_git_sha() {
+  echo $(git rev-parse --short HEAD)
+}
+current_head_commit() {
+  echo $(git log -1 --format=%H)
+}
+other_branches_same_commit() {
+  CURRENT_BRANCH=$(current_branch)
+  CURRENT_COMMIT=$(current_head_commit)
+  REMOTE_BRANCH_PREFIX='refs/remotes/origin/'
+
+  git show-ref |  #list all references in current rep
+  grep $CURRENT_COMMIT |  #find current commit sha
+  grep -Eo "$REMOTE_BRANCH_PREFIX.*" |  #keep only remote branches
+  cut -c "$((${#REMOTE_BRANCH_PREFIX} + 1))-" | #remove prefixes
+  grep -Ev "^(${CURRENT_BRANCH})|(HEAD)"  #exclude the current working branch
+}
```

Comment:
This ought to allow us to see what rc_* or hotfix_* branch was deployed to test, staging or prod.